aten/src/ATen/native/native_functions.yaml

```diff
@@ -4281,9 +4281,13 @@
   dispatch:
     CompositeExplicitAutograd: rot90
 
+- func: trapezoid.x(Tensor y, Tensor x, *, int dim=-1) -> Tensor
+
+- func: trapezoid.dx(Tensor y, *, Scalar dx=1, int dim=-1) -> Tensor
+
 - func: trapz.x(Tensor y, Tensor x, *, int dim=-1) -> Tensor
 
-- func: trapz.dx(Tensor y, *, float dx=1, int dim=-1) -> Tensor
+- func: trapz.dx(Tensor y, *, float dx=1.0, int dim=-1) -> Tensor
```

Comment:
This might trigger a backwards compatibility check warning

Reply:
I will un-do this.